code/online_challenges/src/codechef/HORSES/HORSES.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+
+#include<stdlib.h>
+
+int main(void) 
+{
+    int t, i, j, k, n;
+    scanf("%d", & t);
+    for (i = 1; i <= t; i++) 
+    {
+        scanf("%d", & n);
+        int s[n];
+        for (j = 0; j < n; j++)
+            scanf("%d", & s[j]);
+        int min = abs(s[0] - s[1]);
+        for (k = 0; k < n; k++) {
```

Comment:
This variable should be called j as well and the one inside k. The declaration of k should be above the innerr loop. Also use pre-increment whenever possible.